CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# 2.0.0
+
+## Added
+
+* Added `CHANGELOG.md` and `MODULE_CHANGELOG.md` for version control.
+
+## Changed
+
+* Updated `appveyor.yml` to use latest pattern from `Whiskey`.
+* Updated the `Version`, `Zip`, `CopyFile` and `Pester` task in `whiskey.yml`.
+* Updated `README.md` to reflect these changes.
+* Updated `Initialize-Repository.ps1` to rename `MODULE_CHANGELOG.md` to `CHANGELOG.md`
+
+## Removed
+
+* Removed `Publish` pipeline and its tasks from `whiskey.yml`.

```

Comment:
Change to `* Removed all publishing tasks and logic from the default whiskey.yml. Publishing is now recommended to be done by AppVeyor.`